src/protocol.cpp

```diff
@@ -82,6 +82,8 @@ const char *CLSIG="clsig";
 const char *ISLOCK="islock";
 const char *ISDLOCK="isdlock";
 const char *MNAUTH="mnauth";
+const char *GETQUORUMROTATIONINFO="getquorumrotationinfo";
```

Comment:
@ogabrielides I wouldn't have expected you to know this and didn't think about it until debugging, but p2p command names are [limited to 12 characters](https://github.com/dashpay/dash/blob/develop/src/protocol.h#L32). Thus the bizarrely truncated names of some of the other messages :slightly_smiling_face: 
Not sure what it should be name: `getqrinfo` and `qrinfo` 

@PastaPastaPasta @UdjinM6 please comment (and correct if this is not accurate).